pyiron_atomistics/atomistics/master/phonopy.py

```diff
@@ -32,21 +32,23 @@
 s = Settings()
 
 
-class thermal(object):
+class Thermal:
     """
+    Holds thermal properties that are the results of the phonopy calculation.
 
     Args:
-        temps:
-        fe:
-        entropy:
-        cv:
+        temperatures (ndarray): temperatures at which the other quantities are evaluated, units of Kelvin
+        free_energies (ndarray): free energies in the quasi-harmonic approximation from phonon contributions, units of
+                                 electron volts
+        entropy (ndarray): vibrational entropy calculated from the above free energy
+        cv (ndarray): heat capacity at constant volume, units of kJ/K/mol
     """
 
-    def __init__(self, temps, fe, entropy, cv):
+    def __init__(self, temperatures, free_energies, entropy, cv):
         KJ_mol_to_eV = 0.01036410
```

Comment:
@pmrv Can you replace the float number with the corresponding scipy constant ? - I know it is not part of this pull request, but we should try to remove magical numbers when ever we spot them.